src/cf-layout.less

```diff
@@ -1605,6 +1605,121 @@
     }
 }
 
+/* topdoc
+  name: Featured content module
+  family: cf-layout
+  patterns:
+    - name: .featured-content-module
+      markup:
+        <section class="block
+                        block__border
+                        block__flush
+                        featured-content-module">
```

Comment:
I think holding off on that makes sense since we're probably not ready to go down the preparing for modular design rabbit hole in Capital Framework yet. We're still learning and prototyping and I don't think it's ready to start moving over yet.